packages/testcontainers/src/container-runtime/image-name.test.ts

```diff
@@ -96,5 +116,13 @@ describe("ContainerImage", () => {
       expect(imageName.image).toBe("image");
       expect(imageName.tag).toBe("sha256:1234abcd1234abcd1234abcd1234abcd");
     });
+
+    it("should work with image being an image ID", () => {
```

Comment:
Similarly to `should keep other tags (not latest) on image IDs`, do we need a `fromString` test where the input has a non-latest tag (`aa285b773a2c042056883845aea893a743d358a5d40f61734fa228fde93dae6f:1`)?

Reply:
Yes, that's probably a good idea. I've added a commit with a test (and also added the docker error message to the text explaining that it won't accept the reference).